content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_addon_coredns.md

```diff
@@ -0,0 +1,31 @@
+
+Installs the CoreDNS addon to a Kubernetes cluster
```

Comment:
Please do not change this file manually. It is supposed to be generated automatically from upstream tools.